sdk/eventhubs/azure-messaging-eventhubs/test/eventhubs-stress-test/src/eventhubs_stress_test.cpp

```diff
@@ -88,8 +178,11 @@ void InitLogger()
     // Create OTLP exporter instance
     auto exporter = otlp::OtlpHttpLogRecordExporterFactory::Create(logger_opts);
     auto processor = logs_sdk::SimpleLogRecordProcessorFactory::Create(std::move(exporter));
+
+    auto resource{GetTraceResource(stressScenarioName)};
+
     std::shared_ptr<logs::LoggerProvider> provider
-        = logs_sdk::LoggerProviderFactory::Create(std::move(processor));
+        = logs_sdk::LoggerProviderFactory::Create(std::move(processor), std::move(resource));
```

Comment:
are you sure about this std::move? why not have the move in the method called / constructor?

Reply:
I don't own the method called - it's a part of the opentelemetry SDK. 

And this is the pattern used by all the OpenTelemetry samples and documentation.